Test/FMI3/fmi3_xml_parsing_test.c

```diff
@@ -163,13 +157,10 @@ void test_variable_naming_conventions(void)
 
     /* list of variables */
     fail_name_check("naming_conventions_xmls/list/aemptyc");
-    g_expected_message = "Two variables with the same name";
     pass_name_check("naming_conventions_xmls/list/cba");
-    fail_name_check("naming_conventions_xmls/list/acad");
```

Comment:
Now tested in variable_test

Reply:
Yeah, seems okay to remove this now duplicated check.